### PR TITLE
Return to opening time

### PR DESCRIPTION
**Pull request by james2m**
Complementary opposite method to the recent advance_to_closing_time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,16 +63,26 @@ WorkingHours.advance_to_closing_time(Time.utc(2014, 8, 4, 7, 16)) # => Mon, 04 A
...
+monday = WorkingHours.next_working_time(sunday) # => Mon, 04 Aug 2014 09:00:00 UTC +00:00
+tuesday = WorkingHours.next_working_time(monday) # => Tue, 05 Aug 2014 09:00:00 UTC +00:00
+
 # Return to previous working time
 WorkingHours.return_to_working_time(Time.utc(2014, 8, 4, 7, 16)) # => Fri, 01 Aug 2014 17:00:00 UTC +00:00
+
+# Return to last opening time
+WorkingHours.return_to_closing_time(Time.utc(2014, 8, 5, 7, 16)) # => Mon, 04 Aug 2014 09:00:00 UTC +00:00
+WorkingHours.return_to_closing_time(Time.utc(2014, 8, 4, 10, 16)) # => Mon, 04 Aug 2014 09:00:00 UTC +00:00
+WorkingHours.return_to_closing_time(Time.utc(2014, 8, 4, 18, 16)) # => Tue, 05 Aug 2014 09:00:00 UTC +00:00
```
Comment (jarthod) [inline README.md]: `return_to_opening_time` I guess ?

**Comment by jarthod** [comment]
Sorry for the delay, here is my review:
You should refactor `return_to_opening_time` to use `return_to_working_time` first to avoid duplicate logic. Also, the `windows_between` method, which is quite complex, is not tested at all. Please add specs for this or keep it for another pull request.
There's also the little typo I commented in the readme.
Thanks !

**Comment by james2m** [comment]
No problem. The windows_between was an accidental push, sorry about that. It was a bit of a spike but is all working on another branch. 

I've refactored return_to_opening_time I'll re-submit when I've extracted it from the other branch. It may be 2 weeks as I'm totally maxed out until then. I've also got a bunch of stuff to stop loops and errors when moving in and out of dst, a bit late for this year, but I think it needs addressing so I will try to assemble a complete and tested pull request for that.

**Comment by dmitry** [comment]
Would be great to see that merged!

**Comment by jarthod** [comment]
@dmitry the PR has never been amended after my review so it is not ready to merge.

**Comment by dmitry** [comment]
@jarthod I see, but might be @james2m will have some time to complete it. :)

**Comment by jarthod** [comment]
Maybe ;) if you're also interested in this, feel free to take @james2m changes and create a pull request yourself with comments addressed.

**Pull request closed without merging**